tox.ini

```diff
@@ -1,80 +1,83 @@
 [tox]
-envlist =
-        py38-celery{50}-django{32, 40}-drflatest
-        quality,
-        version_check,
-        pii_check,
-        translations
+envlist = 
+    py38-celery{50}-django{32,40,42}-drflatest
+    quality,
+    version_check,
+    pii_check,
+    translations
+
 [testenv]
-deps =
+deps = 
     django32: Django>=3.2,<3.3
     django40: Django>=4.0,<4.1
+    django42: Django>=4.2,<4.3
     drflatest: djangorestframework
     celery50: -r{toxinidir}/requirements/celery50.txt
     -rrequirements/test.txt
-commands =
+commands = 
     python -Wd -m pytest {posargs:-n 3}
 
 [testenv:js_tests]
-whitelist_externals =
+whitelist_externals = 
     npm
     make
-commands =
+commands = 
     npm install -g gulp-cli
     npm install
     make test-js
 
 [testenv:js_lint]
-whitelist_externals =
+whitelist_externals = 
     npm
     make
-commands =
+commands = 
     npm install -g gulp-cli
     npm install
     make lint-js
 
 [testenv:rst_validation]
-whitelist_externals =
+whitelist_externals = 
     make
-deps =
+deps = 
     -r{toxinidir}/requirements/quality.txt
-commands =
+commands = 
     make quality-rst
 
 [testenv:quality]
-whitelist_externals =
+whitelist_externals = 
     make
     rm
     touch
-deps =
+deps = 
     -r{toxinidir}/requirements/quality.txt
     -r{toxinidir}/requirements/test.txt
-commands =
+commands = 
     pylint edx_proctoring
     pycodestyle edx_proctoring
     isort --check-only --diff edx_proctoring manage.py setup.py
 
 [testenv:version_check]
-deps =
+deps = 
     -r{toxinidir}/requirements/base.txt
-commands =
+commands = 
     {toxinidir}/edx_proctoring/scripts/version_check.py
 
 [testenv:pii_check]
-whitelist_externals =
+whitelist_externals = 
     make
-deps =
+deps = 
     -r{toxinidir}/requirements/test.txt
     Django>=3.2,<3.3
-commands =
+commands = 
     make pii_check
 
 [testenv:translations]
-whitelist_externals =
+whitelist_externals = 
     make
-deps =
+deps = 
```

Comment:
Update deps to install custom django version for testing 
```
    django32: Django>=3.2,<3.3
    django42: Django>=4.2,<4.3
```

Reply:
done with comment.